grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractUnframedGrpcService.java

```diff
@@ -218,7 +224,10 @@ public void onSubscribe(Subscription subscription) {
             @Override
             public void onNext(DeframedMessage message) {
                 // We know that we don't support compression, so this is always a ByteBuf.
-                final HttpData unframedContent = HttpData.wrap(message.buf()).withEndOfStream();
+                HttpData unframedContent = HttpData.wrap(message.buf()).withEndOfStream();
+                if (responseBodyConverter != null) {
+                    unframedContent = responseBodyConverter.apply(unframedContent);
```

Comment:
Should we also call `.withEndOfStream()` for `unframedContent`?

Reply:
Was this resolved? I don't see any `.withEndOfStream()` call.

Reply:
Yes sir, according to @ikhoon , After https://github.com/line/armeria/pull/3956 was applied, we don't need to call .withEndOfStream() anymore.

Reply:
Oh, didn't realize that. Never mind, then.